CHANGELOG.md

```diff
@@ -1,90 +1,127 @@
 # Changelog
+
 ## [Unreleased]
 
+## [1.6.1] - 2020-07-03
+
+- Updated graphql peer dependency to ^16
+
 ## [1.6.0] - 2020-07-03
+
```

Comment:
Could you please remove all the empty lines.

Reply:
Yep! Sorry about that!

Reply:
Done.